Sources/Jobs/Scheduler/JobSchedule.swift

```diff
@@ -137,7 +137,7 @@ public struct JobSchedule: MutableCollection, Sendable {
 
     /// AsyncSequence of Jobs based on a JobSchedule
     struct JobSequence: AsyncSequence {
-        typealias Element = (job: JobParameters, date: Date)
+        typealias Element = (job: JobParameters, date: Date, nextScheduledAt: Date?)
```

Comment:
Perhaps we should convert this into a struct now, given the number of parameters is increasing
```suggestion
        struct Element {
            let job: JobParameters
            let date: Date
            let nextScheduledAt: Date?
        }
```

Reply:
Sounds good. I will address these in the next commit.